client/packages/core/src/Reactor.js

```diff
@@ -1102,20 +1106,24 @@ export default class Reactor {
     }
     this._log.info('[socket][open]', this._ws._id);
     this._setStatus(STATUS.OPENED);
-    this.getCurrentUser().then((resp) => {
-      this._trySend(uuid(), {
-        op: 'init',
-        'app-id': this.config.appId,
-        'refresh-token': resp.user?.['refresh_token'],
-        versions: this.versions,
-        // If an admin token is provided for an app, we will
-        // skip all permission checks. This is an advanced feature,
-        // to let users write internal tools
-        // This option is not exposed in `Config`, as it's
-        // not ready for prime time
-        '__admin-token': this.config.__adminToken,
+    this.getCurrentUser()
+      .then((resp) => {
+        this._trySend(uuid(), {
+          op: 'init',
+          'app-id': this.config.appId,
+          'refresh-token': resp.user?.['refresh_token'],
+          versions: this.versions,
+          // If an admin token is provided for an app, we will
+          // skip all permission checks. This is an advanced feature,
+          // to let users write internal tools
+          // This option is not exposed in `Config`, as it's
+          // not ready for prime time
+          '__admin-token': this.config.__adminToken,
+        });
+      })
+      .catch((e) => {
+        this._log.error('[socket][error]', targetWs._id, e);
```

Comment:
Added this logging, as Reactor was failing silently. In this case `getCurrentUser` failed, but the error was swallowed